MethodSwitch 1

froschs opened the pull request:
First publish of the MethodSwitch program.

froschs committed:
MethodSwitch 1

First publish of the MethodSwitch program.

steve-plas commented:
The only thing I noticed, is you forgot to do the comments on your functions by using the /*\* "enter" that he showed in class this week. 
But other than that it looks like nice and clean code, and I like the use of the switch statement.

froschs committed:
MethodSwitch 2

Added comments to the MethodSwitch program.

froschs commented:
Thanks Steven. Got carried away with formatting what became some pretty sloppy code and forgot all about comments.

srtinkey commented on MethodSwitch/src/methodswitch/MethodSwitch.java:
```diff
@@ -0,0 +1,170 @@
+        System.out.println("Enter '1' to play FIZZBUZZ.\n" + "Enter '2' to view the value of your chosen word.\n" + "Enter '3' to use the salary calculator.\n"
+                + "Enter '4' to check your grade.\n" + "Enter '5' to play Truth or Dare.\n" + "Enter '6' or any other number to EXIT.");
+        number = keyboard.nextInt();
+        
```
If each menu item were on a separate line, it would make reading and editing it much easier.

srtinkey commented:
Nice program, neat and easy to read (except where noted).

froschs committed:
MethodSwitch 3

Changed formatting in the MethodSwitch program to make menu items easier
to read.

ninjapanzer commented on MethodSwitch/src/methodswitch/MethodSwitch.java:
```diff
@@ -0,0 +1,174 @@
+        System.out.println("Enter 'Truth' or 'Dare':");
+        response = keyboard.next();
+                
+        if (response.equalsIgnoreCase("TRUTH")){
```
These truth or dare literals are perfect opportunities to use constants within your function.

``` java
final String TRUTH = "truth";
final String DARE = "dare";

if(response.equalsIgnoreCase(TRUTH)){}
if(response.equalsIgnoreCase(DARE)){}
```

No requirement to make this change just a way of thinking about it. I felt this `"TRUTH"` variable was almost screaming out like you wanted it to be a constant :+1:

ninjapanzer commented on MethodSwitch/src/methodswitch/MethodSwitch.java:
```diff
@@ -0,0 +1,174 @@
+    
+    /**
+     * Takes a user inputted number which if divisible by 3 outputs fizz,
+     * by 5 buzz, and both fizzbuzz. Neither gives an invalid statement.
```
the annotations you are looking for are `@param` and `@return`

ninjapanzer commented on MethodSwitch/src/methodswitch/MethodSwitch.java:
```diff
@@ -0,0 +1,174 @@
+     */
+    public static void fizzBuzz(){
+        int fbNumber;
+        Scanner keyboard = new Scanner(System.in);
```
Albeit repetitive I do like how you have a new instance of scanner in each function. The `new` operator is cheap and its ok to use it over and over again. More on that to come in class.
